flink-tests/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterStopWithSavepointITCase.java

```diff
@@ -81,6 +82,13 @@
 
     @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
+    @Rule
+    public Timeout timeout =
+            Timeout.builder()
+                    .withTimeout(15, TimeUnit.SECONDS)
```

Comment:
Make the timeout really big. It doesn't matter if it takes 1 or 10 min on AZP to fail as long as it fails.
For manual tests, you adjust it anyways.
 
In general, timeouts on ITCases are quite error-prone on AZP when a heavy test is also executed at the same time. I'd probably more go into the direction of having no timeouts on AZP but just the general timeout.

Reply:
Makes sense. Would you recommend removing the timeout whatsoever then, assuming we can get: https://github.com/apache/flink/pull/15546 soonish?

Reply:
Yes.